roles/foreman_repositories/tasks/redhat.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: 'Install centos-release-scl-rh' # noqa 403
+  yum:
+    name: centos-release-scl-rh
+    state: latest
+    update_cache: yes
+  tags:
+    - packages
+  when:
+    - ansible_distribution == 'CentOS'
+    - ansible_distribution_major_version == '7'
+
+- name: 'Setup Foreman {{ foreman_repositories_version }} Repository'
+  yum:
+    name: https://yum.theforeman.org/releases/{{ foreman_repositories_version }}/el{{ ansible_distribution_major_version }}/x86_64/foreman-release.rpm
+    disable_gpg_check: True
+    state: present
+  tags:
+    - packages
+
+- name: 'Setup Katello {{ katello_repositories_version }} Repository'
+  yum:
+    name: https://fedorapeople.org/groups/katello/releases/yum/{{ foreman_repositories_katello_version }}/katello/el{{ ansible_distribution_major_version }}/x86_64/katello-repos-latest.rpm # noqa 204
+    disable_gpg_check: True
+    state: present
+  when: foreman_repositories_katello_version
```

Comment:
`when: foreman_repositories_katello_version is defined`? then you can omit it from the defaults file?